client/ayon_core/tools/utils/widgets.py

```diff
@@ -104,6 +105,151 @@ def __init__(self, *args, **kwargs):
             self.setPalette(filter_palette)
 
 
+def get_down_arrow_icon() -> QtGui.QIcon:
+    normal_pixmap = QtGui.QPixmap(
+        get_style_image_path("down_arrow")
+    )
+    on_pixmap = QtGui.QPixmap(
+        get_style_image_path("down_arrow_on")
+    )
+    disabled_pixmap = QtGui.QPixmap(
+        get_style_image_path("down_arrow_disabled")
+    )
```

Comment:
I wonder if it'd make sense to make these image paths constants so we don't need to check the file's existence each time?

Reply:
Done.